integration-tests/tests/src/node/node-fetch.ts

```diff
@@ -16,15 +16,20 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
-export * from "../index";
+import { expect } from "chai";
+import nodeFetch from "node-fetch";
+import Realm from "realm";
 
-import { DefaultNetworkTransport } from "../DefaultNetworkTransport";
-import { Fetch, AbortController } from "../types";
-import { makeRequestBodyIterable } from "../IterableReadableStream";
+import { importAppBefore } from "../hooks";
+import { buildAppConfig } from "../utils/build-app-config";
+import { baseUrl } from "../utils/import-app";
 
-import fetch from "node-fetch";
-import NodeAbortController from "abort-controller";
+describe.skipIf(environment.missingServer, "passing node-fetch to AppConfiguration", () => {
+  importAppBefore(buildAppConfig().anonAuth());
 
-DefaultNetworkTransport.fetch = fetch as Fetch;
-DefaultNetworkTransport.AbortController = NodeAbortController as AbortController;
-DefaultNetworkTransport.transformResponse = makeRequestBodyIterable;
+  it("is supported", async function (this: AppContext) {
+    const app = new Realm.App({ id: this.app.id, baseUrl, fetch: nodeFetch });
+    const user = await app.logIn(Realm.Credentials.anonymous());
+    expect(typeof user.id).equals("string");
```

Comment:
I could probably just await the login, instead of actually expecting anything of the result 🤔

The test doesn't actually verify that the `fetch` provided will be used, I added another (more general) test for that. The main purpose of this is actually that I wanted to test that the type for the `fetch` option would accept the `node-fetch` export.

Reply:
Yup I agree, the expect is likely not necessary for the test 👍